projects/plugins/super-cache/tests/e2e/specs/default-settings.test.ts

```diff
@@ -13,21 +20,25 @@ describe( 'cache behavior with default settings', () => {
 		} );
 	} );
 
+	beforeEach( async () => {
+		await deleteCacheDirectory();
+	} );
+
 	test( 'caches URLs with no get parameters', async () => {
 		const first = await loadPage();
 		const second = await loadPage();
 
 		expect( first ).toBe( second );
 	} );
 
-	test( 'logged in users do not get cached pages', async () => {
+	test( 'logged in users get cached pages', async () => {
 		const cookie = await getAuthCookie();
 		const url = getSiteUrl();
 
 		const first = await authenticatedRequest( cookie, 'GET', url );
 		const second = await authenticatedRequest( cookie, 'GET', url );
 
-		expect( first ).not.toBe( second );
+		expect( first ).toBe( second );
```

Comment:
I am a bit confused about this test. Isn't it the default behavior not to cache logged-in users?

Reply:
It depends on how you turn Super Cache on. If you turn it on via the "Easy" tab, then yes it sets that setting by default. If you turn it on via the Advanced tab, it doesn't change any settings - including leaving caching set to "Enable caching for all visitors."